connectivity/drivers/802.15.4_RF/atmel-rf-driver/CMakeLists.txt

```
@@ -1,15 +1,19 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-target_include_directories(mbed-os
-    PUBLIC
+add_library(mbed-os-802.15.4-atmel-rf INTERFACE)
```

```suggestion
add_library(mbed-os-802.15.4-atmel INTERFACE)
```

It is implied in because of `802.15.4` and is in line with others you have created (`i.e mbed-os-802.15.4-stm-s2lp`).